### ISSUE TITLE
Compatibility break with core 0.19.x

### ISSUE THREAD

**Issue by universalhandle** [curveball/aws-lambda#35]
`aws-lambda` v0.2.2 doesn't work with `core` v0.19.0.

This simple application:
```ts
import { Application } from '@curveball/core';
import problemMiddleware from '@curveball/problem';
import lambdaHandler from '@curveball/aws-lambda';

const app = new Application();
// order of middlewares matters; "problem" should be at or near the top for error handling
app.use(problemMiddleware());
app.use((ctx) => {
  ctx.response.body = 'Hello, cruel world!';
});

export const handler = lambdaHandler(app);
```

... compiles but yields runtime errors:

```
✖ Unhandled exception in handler 'webhook'.
✖ Cannot convert undefined or null to object
✖ TypeError: Cannot convert undefined or null to object
      at Function.keys (<anonymous>)
      at new Headers (/home/universalhandle/Code/sample-project/node_modules/@curveball/core/dist/headers.js:7:34)
      at new MemoryRequest (/home/universalhandle/Code/sample-project/node_modules/@curveball/core/dist/memory-request.js:14:28)
      at /home/universalhandle/Code/sample-project/node_modules/@curveball/aws-lambda/dist/index.js:9:25
      at InProcessRunner.run (file:///home/universalhandle/Code/sample-project/node_modules/serverless-offline/src/lambda/handler-runner/in-process-runner/InProcessRunner.js:87:20)
      at async MessagePort.<anonymous> (file:///home/universalhandle/Code/sample-project/node_modules/serverless-offline/src/lambda/handler-runner/worker-thread-runner/workerThreadHelper.js:25:14)
```

`aws-lambda` states a peer dependency on `core` versions `>=0.16 <1`. Potential fixes:
- short term: change the peer deps to be `>0.16 <0.19`
- longer term: identify the breaking change in `core` 0.19.0 and patch it

Obviously the second fix is the "right" fix, but I haven't investigated it yet; it took quite a bit of debugging to get just this far. I figured there was a chance you'd know right away what the problem is, so I decided to open this before digging any deeper.

**Comment by evert** [comment]
I released both 0.2.3 and 0.3.0. The former has compatibility with curveball < 0.19 and the latter => 0.19. Hope this fixes your issue!

**Comment by universalhandle** [comment]
Perfect. Thanks!
